homework-submission/week1/src/index.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import './index.css';
+// import App from './App';
+import * as serviceWorker from './serviceWorker';
+import TodoList from "./todo";
+
+ReactDOM.render(<TodoList />, document.getElementById('root'));
```

Comment:
It's better to stick to "<App />" and within the App.js you can add whatever component you like.

The reason behind this is, There should be a single main "parent" component for all of your upcoming new features, also you would use App component for adding Router and Routes in future classes.